include/mysql/plugin.h

```diff
@@ -143,6 +143,9 @@ enum enum_mysql_show_type
   SHOW_INT,        ///< shown as _unsigned_ int
   SHOW_LONG,       ///< shown as _unsigned_ long
   SHOW_LONGLONG,   ///< shown as _unsigned_ longlong
+  SHOW_SIGNED_INT,
+  SHOW_SIGNED_LONG,
```

Comment:
Duplicate with the one in sql_plugin_enum.h. Probably that one has to be removed.